e2e_tests/standard/test_files.py

```diff
@@ -121,3 +125,168 @@ def test_fileset(self, container: AttributeContainer):
                         content = file.read()
                         assert len(content) == len(expected_content)
                         assert content == expected_content
+
+    @staticmethod
+    def _gen_possible_paths():
```

Comment:
Replace it with faker file_path :wink: https://faker.readthedocs.io/en/master/providers/faker.providers.file.html#faker.providers.file.Provider.file_path